src/wavesurfer.js

```diff
@@ -1022,10 +1022,12 @@ export default class WaveSurfer extends util.Observer {
      * Loads audio and re-renders the waveform.
      *
      * @param {string} url The url of the audio file
-     * @param {?number[]|number[][]} peaks Wavesurfer does not have to decode the audio to
-     * render the waveform if this is specified
+     * @param {?number[]|number[][]} peaks Wavesurfer does not have to decode
+     * the audio to render the waveform if this is specified
      * @param {?string} preload (Use with backend `MediaElement`)
      * `'none'|'metadata'|'auto'` Preload attribute for the media element
+     * @param {?number} duration The duration of the audio. This is required to
```

Comment:
I'm not a fan of the word `required` here as this is an optional thing. Could you rephrase this a little?

Reply:
Done. I've tried to explain it a little more in the docs and removed the param docs on the private methods for brevity.